.github/workflows/foreman_plugin.yml

```diff
@@ -85,6 +90,8 @@ jobs:
         include: ${{ fromJson(needs.setup_matrix.outputs.matrix).include }}
         exclude: ${{ fromJson(needs.setup_matrix.outputs.matrix).exclude }}
     steps:
+      - run: |
+          echo "${{ inputs.environment_variables }}" >> .env.caller; cat .env.caller >> $GITHUB_ENV; rm .env.caller
```

Comment:
echo directly into GITHUB_ENV doesn't work?

Reply:
It does. I've changed the line accordingly